src/convertor/OndcConvertor.spec.js

```diff
@@ -0,0 +1,80 @@
+import {
+  it, expect, describe,
+} from 'vitest';
+import OndcConvertor from './OndcConvertor';
+import platformMapperConfig from '../resource/test/expectedMapperOutput.json';
+import OndcItemMapper from './OndcItemMapper';
+
+describe('OndcConvertor', () => {
+  it('should convert platform item id to ondc item id ', async () => {
+    const config = [
+      {
+        ondc: 'id',
+        ondcDataType: 'string',
+        platform: '.id',
+        platformDataType: 'number',
+      }];
+    const ondcMapper = new OndcItemMapper(config);
+    const ondcMatchedTags = await ondcMapper.map({
+      id: 52,
+    });
+    const convertedOndcResponse = await OndcConvertor.convert(ondcMatchedTags);
```

Comment:
This is a unit test. We test unit of work - like This file should test only OndcConvertor. You cannot combine OndcItemMapper functionality with OndcConvertor

Reply:
updated